index.html

```diff
@@ -14,7 +14,7 @@
     <div class="text-center">
     <p class="lead">Apache Druid is a real-time database to power modern analytics applications.</p>
     <p><a class="button" href="/downloads.html"><span class="fa fa-download"></span> Download</a>
-    <a class="button" href="https://github.com/apache/druid/"><span class="fab fa-github"></span> GitHub</a></p>
+    <a class="button" href="/community/join-slack?v=1"><span class="fab fa-slack"></span> Join Slack</a></p>
```

Comment:
IMO github and slack are the two most important ways we connect with users. I would like to see both here.

Reply:
<img width="1126" alt="image" src="https://user-images.githubusercontent.com/177816/204402418-b5b4448f-026b-4a36-ac60-b43ba506be77.png">
 Nice, putting 3 there makes Slack in the center which is perfect. (it is secretly the most important one)